### DISCUSSION TITLE
"Pro not being free" is not the point.

### DISCUSSION THREAD

**Discussion by paspo** [rustdesk/rustdesk-server#296] [General]
First of all, we need a better title for this discussion. Any idea is welcome.

My points:
- any opensource project should be "economically" stable, this one included
- it's not bad to have a company behind an open source product (in fact, it makes it more trustworthy in the long term)
- this company should profit in some way on this product
- "pro being free" is nonsense. Pro features should be paid, "community" features should be free.

From these point you can extract that "pro" is not relative to a product, but it's relative to a feature set. You have standard features and pro features.
The bottom line is: if a customer is using rustdesk to add value to his business, he's probably willing to buy pro features, especially if one of these features is direct support (email/phone). 

IMHO the real question is: which feature should be considered "pro" and which feature should be considered "required for this product"?

Take for example the shared address book: you can find that feature in almost all competitors; from the user point of view, this is a **required** feature, something so basic for the product to be considered an alternative to other "big names".
There are other features that are viewed as "mostly useless" if you're not in a business environment: Active Directory integration, 2FA, centralized management/queuing of support connections, branding, deployment, reporting, integration with other software...

I think we should make a list of features (current features AND ideas) and catalog everything as "must-have", "business only", "middle ground". The last one is really important because it can help identify something that distinguish rustdesk from the competitor, and that something can be either left free to acquire more users or can be a "pro" feature, if nobody else has it (and it is interesting for a business).

This helps on comparing rustdesk to competitors also. 
And having such a list in plain sight will tell a lot about the "openness" of your company.

**Comment by fengyuclub** [comment]
I very much agree with the above point of view, and hope that rustdesk will get better and better

**Comment by TheUntouchable** [comment]
As you mentioned, Rustdesk without the address book is really not an alternative for a normal, non profit user, as all other products include it for free and the price for it is just too high.

EDIT: Btw Rustdesk guys, you should really have a look at your pricing site.. Otherwise you have to pay me for using your software :D
![grafik](https://github.com/rustdesk/rustdesk-server/assets/2162540/e72b0584-8ac0-4b71-a6d4-9a3cbab6f444)

**Comment by Kurgan-** [comment]
Actually I could also consider paying for the pro version, but the cost is too high, in my opinion, at least for the basic one-user plan. For a single user I currently use Anydesk (which has been breached just yesterday, that's why I'm here now) at a price of about 70 euros per year. I have one user but I can connect to unlimited remote devices (not only 20), and I also get them to host the servers. 
I'm perfectly fine hosting my own server, which is even more secure than using "the cloud", but then I have to consider the cost of the server, too.

If Rustdesk pro worked as good as anydesk, allowed for unlimited remote connections and not only 20, and was cheaper, in the 50-60 euros per year, then I'd be interested in it.
